README.md

```diff
@@ -61,11 +61,19 @@ Add the following line to your `.zshrc`:
 
     antigen bundle soimort/you-get
 
-### Option 3: Use a pre-built package (Windows only)
+### Option 3: Using [Chocolatey](https://chocolatey.org/) (Windows only)
+
+    choco install you-get
+
+The chocolatey package source can be found [here](https://github.com/chantisnake/you-get-choco)
+
+The chocolatey package page can be found [here](https://chocolatey.org/packages/you-get/0.4.486)
```

Comment:
Don't use any fixed version number (i.e. `0.4.486`). Actually, these two links seem unnecessary, since the command `choco install you-get` has covered everything a user should need.